README.md

```diff
@@ -30,7 +30,7 @@ and writers for Parquet files.
 
 We created Parquet to make the advantages of compressed, efficient columnar data representation available to any project in the Hadoop ecosystem.
 
-Parquet is built from the ground up with complex nested data structures in mind, and uses the [record shredding and assembly algorithm](https://github.com/Parquet/parquet-mr/wiki/The-striping-and-assembly-algorithms-from-the-Dremel-paper) described in the Dremel paper. We believe this approach is superior to simple flattening of nested name spaces.
+Parquet is built from the ground up with complex nested data structures in mind, and uses the [record shredding and assembly algorithm](https://github.com/julienledem/redelm/wiki/The-striping-and-assembly-algorithms-from-the-Dremel-paper) described in the Dremel paper. We believe this approach is superior to simple flattening of nested name spaces.
```

Comment:
The Wiki page hosted under `julienledem/redelm` is the only valid online copy I could find right now. Maybe we want to migrate this to the Apache Wiki.